players/management/commands/generate_players.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+import os
+import uuid
+
+from django.core.management import BaseCommand
+from django.test import RequestFactory
+from importlib import import_module
+from django.conf import settings
+from django.contrib.auth.models import User
+from players.views import code as code_view
+
+
+_AVATAR_CODES_DIRECTORY = 'players/avatar_examples'
```

Comment:
Should be relative to this file otherwise cannot run command unless in the root directory.